src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralBrowseObject.xaml

```diff
@@ -49,6 +49,113 @@
       <DataSource Persistence="ProjectFile" PersistedName="Win32Resource" />
     </StringProperty.DataSource>
   </StringProperty>
+    <StringProperty Name="DefineConstants" DisplayName="Define Constants">
```

Comment:
So create a CsharpBrowseObject.xml / VisualStudioBrowseObject.xml / CompilerBrowseObject.xml and any thing shared in GeneralBrowseObject.xml ?